docsite/posts/config/env_vars.md

```diff
@@ -59,11 +59,14 @@ These settings control the text generation model used by the pipeline. Any setti
 | `GRAPHRAG_LLM_THREAD_COUNT`                       |                          | The number of threads to use for LLM parallelization.                          | `int`   | 50                    |
 | `GRAPHRAG_LLM_THREAD_STAGGER`                     |                          | The time to wait (in seconds) between starting each thread.                    | `float` | 0.3                   |
 | `GRAPHRAG_LLM_CONCURRENT_REQUESTS`                |                          | The number of concurrent requests to allow for the embedding client.           | `int`   | 25                    |
-| `GRAPHRAG_LLM_TPM`                                |                          | The number of tokens per minute to allow for the LLM client. 0 = Bypass        | `int`   | 0                     |
-| `GRAPHRAG_LLM_RPM`                                |                          | The number of requests per minute to allow for the LLM client. 0 = Bypass      | `int`   | 0                     |
+| `GRAPHRAG_LLM_TOKENS_PER_MINUTE`                  |                          | The number of tokens per minute to allow for the LLM client. 0 = Bypass        | `int`   | 0                     |
```

Comment:
How do we want to characterize param changes like this? Seems breaking (e.g., major bump) unless we build in a fallback to the old one for back-compat

Reply:
@darthtrevino ?

Reply:
If we're pre-1.0, then we can release this as a minor version bump. The fallbacks are useful for expected env-vars like `OPENAI_API_KEY`, but for stuff like this I think we should just change it over. The fallbacks introduce some complexity.

Reply:
And, another thing here is that RPM and TPM were never working, because it was named in one way and parsed in another one, so I don't thing we would need a fallback.